packages/kbn-es/src/install/archive.js

```diff
@@ -44,13 +44,19 @@ exports.installArchive = async function installArchive(archive, options = {}) {
     log = defaultLog,
   } = options;
 
+  let dest = archive;
+  if (archive.startsWith('http')) {
```

Comment:
I think maybe we want something a tiny bit more robust, maybe `['http:', 'https:'].includes(url.parse(archive).protocol)`?

Reply:
makes sense